src/Generation/GapicClientGenerator.php

```diff
@@ -361,25 +362,34 @@ private function lroMethods(): Vector
 
     private function getClientDefaults(): PhpClassMember
     {
+        $clientDefaultValues = [
+        'serviceName' => AST::access(AST::SELF, $this->serviceName()),
+        // TODO: Understand if this should be named 'serviceAddress' or 'apiEndpoint'?
+        'serviceAddress' => AST::concat(AST::access(AST::SELF, $this->serviceAddress()), ':', AST::access(AST::SELF, $this->servicePort())),
+        'clientConfig' => AST::concat(AST::__DIR__, "/../resources/{$this->serviceDetails->clientConfigFilename}"),
+        'descriptorsConfigPath' => AST::concat(AST::__DIR__, "/../resources/{$this->serviceDetails->descriptorConfigFilename}"),
+      ];
+
+        // TODO: Consolidate setting all the known array values together.
+        // We do this here to maintain the existing sensible ordering.
+        if ($this->serviceDetails->transportType === Transport::GRPC_REST) {
+            $clientDefaultValues['gcpApiConfigPath'] =
+          AST::concat(AST::__DIR__, "/../resources/{$this->serviceDetails->grpcConfigFilename}");
+        }
+
+        $clientDefaultValues['credentialsConfig'] = AST::array([
+        'defaultScopes' => AST::access(AST::SELF, $this->serviceScopes()),
+      ]);
+        $clientDefaultValues['transportConfig'] = AST::array([
+        'rest' => AST::array([
+          'restClientConfigPath' => AST::concat(AST::__DIR__, "/../resources/{$this->serviceDetails->restConfigFilename}"),
+        ])
+      ]);
```

Comment:
Whitespace is off here too. I'll just make a general comment in my review to fix it everywhere, but 4 spaces and no tabs is our standard.

Reply:
Done.